src/pages/blog.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import styled from 'styled-components'
+import PageContainer from '../components/page-container'
+import SEO from '../components/seo'
+import PostItem from '../components/blogPostPreview'
+import { graphql, useStaticQuery } from 'gatsby'
+
+// const posts = [
+//   {
+//     title: 'Title of post',
+//     body: 'Body text goes here',
+//     date: '01-01-2020',
+//     slug: '/slug',
+//   },
+//   {
+//     title: 'Title of second post',
+//     body: 'Body text goes here.......',
+//     date: '02-01-2020',
+//     slug: '/slug2',
+//   },
+// ]
+
+const Blog = () => {
+  //Contentful plost Posts
+  const contentfulData = useStaticQuery(graphql`
+    query {
+      allContentfulBlogPost(sort: { fields: publishedDate, order: DESC }) {
+        edges {
+          node {
+            title
+            slug
+            publishedDate(formatString: "MMMM Do, YYYY")
+            description
+            body {
+              raw
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  return (
+    <>
+      <SEO title="Blog" description="keep up to date with Human Crafted" />
+      <PageContainer>
+        <Container>
+          <h1>Posts</h1>
+
+          {contentfulData.allContentfulBlogPost.edges.map(postNode => {
+            // console.log(post)
+            return <PostItem post={postNode.node} />
+          })}
```

Comment:
We should consider what would happen if any of `contentfulData.allContentfulBlogPost.edges` are undefined. 
On this line we could do: `          }) || <h1>No Posts</h1>}`